etc/profile-a-l/gtk-lbry-viewer.profile

```diff
@@ -6,7 +6,8 @@ include gtk-lbry-viewer.local
 # added by included profile
 #include globals.local
 
-ignore quiet
+private-bin gtk-lbry-viewer
 
 # Redirect
+include gtk-youtube-viewers-common.profile
 include lbry-viewer.profile
```

Comment:
```suggestion
include gtk-youtube-viewers-common.profile

# Redirect
include lbry-viewer.profile
```

https://github.com/netblue30/firejail/pull/5722#discussion_r1162907061

To be clear, I meant just removing the comment from the first include (while
keeping them separate).